tensorflow/tools/ci_build/Dockerfile.gpu

```diff
@@ -1,7 +1,12 @@
-FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
+FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu14.04
```

Comment:
@gunan:
"As I mentioned before, this governs the package we build for releases.
We cannot build 1.1 release on 16.04 -> so this has to be merged.
Could you point Amit to the changes that need to be done, so we can add them to this PR?"